### ISSUE TITLE
How to support '@3' retina?

### ISSUE THREAD

**Issue by wind4gis** [mixtur/webpack-spritesmith#81]
Excuse me, How to support '@3' retina and '@2' at the same time?

**Comment by mixtur** [comment]
I don't think it is possible with this plugin. It is modeled after [gulp-spritesmith](https://github.com/twolfson/gulp.spritesmith) and it doesn't seem to support that.
It will require different compilation strategy and different api templates. Now it is hard coded to either create spritesheet from one set of images or two spritesheets from two sets of images. And api templates I am using are also assuming wither one or two sets of images.

**Comment by wind4gis** [comment]
@mixtur can I use node-glob, and I put @1 @2 @3 three type of picture into three folder, just new three plugin to support?

**Comment by mixtur** [comment]
You  can create three instances of plugin, yes. It will produce three different spritesheets, depeding on how you configure each instance, but it will also produce three APIs.
If you want them to cooperate you'll have to do this coordination yourself.
